app/scripts/views/components/barChart/barChartTemplate.js

```diff
@@ -41,7 +43,12 @@
           transitionDuration: iViz.opts.dc.transitionDuration
         },
         opts: {},
-        addingChart: false
+        addingChart: false,
+        showDespIcon: true,
+        titleTooltip: {
```

Comment:
Should we move this logic into chartOperationsHeader.js, because all the charts have same implementation.